### PR TITLE
fix: use discord.js-utilities functions

### PR DESCRIPTION
**Pull request by favna**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -17,7 +17,7 @@ services:
         max-file: '3'
 
   postgres:
-    container_name: postgres
```
Comment (kyranet) [inline docker-compose.yml]: What is this change?

**Review by kyranet — CHANGES_REQUESTED** [review]

File: docker-compose.yml [hunk 2]
```diff
@@ -17,7 +17,7 @@ services:
...
+    container_name: postgres2
```
Comment (favna) [inline docker-compose.yml]:
```suggestion
    container_name: postgres
```

File: docker-compose.yml [hunk 3]
```diff
@@ -60,7 +60,7 @@ services:
     container_name: skyra
     depends_on:
       - influx
-      - postgres
+      - postgres2
```
Comment (favna) [inline docker-compose.yml]:
```suggestion
      - postgres
```

File: docker-compose.yml (hunk 1 above)
Comment (favna) [reply docker-compose.yml]: Something I pushed accidentally

**Review by kyranet — APPROVED** [review]

**Pull request merged**